db/drivers/sqlite/db.c

```diff
@@ -29,6 +29,9 @@
  * \return DB_FAILED on error
  * \return DB_OK on success
  */
+
+size_t len;
```

Comment:
It’s not safe to declare a variable for this purpose in file scope. It should be one in each function.
But in this case it seems to to be possible to fit the if statement in one line without a `len` variable.

Reply:
Done. Declared variable inside the functions.